CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## v1.5.2-dev
+
+* Upgrade JS interop code to use extension types
```

Comment:
The previous entry hasn't been published, so we can just combine this entry with that one (we don't need a new `v1.5.2-dev` section).

Reply:
Done